helix-term/tests/integration.rs

```diff
@@ -0,0 +1,31 @@
+#[cfg(feature = "integration")]
+mod test {
+    mod helpers;
+
+    use std::path::PathBuf;
+
+    use helix_core::{syntax::AutoPairConfig, Position, Selection};
+    use helix_term::{args::Args, config::Config};
+
+    use indoc::indoc;
+
+    use self::helpers::*;
+
+    #[tokio::test]
+    async fn hello_world() -> anyhow::Result<()> {
+        test_key_sequence_text_result(
+            Args::default(),
+            Config::default(),
+            ("#[\n|]#", "ihello world<esc>", "hello world#[|\n]#"),
+        )
+        .await?;
+
+        Ok(())
+    }
```

Comment:
Interesting, now we have integration test

Reply:
Hm, I kind of liked the explicitness of seeing that this is a test case with the default config. But yeah, it's seeming like most of the test cases are with the default, so maybe another helper would be good. I kind of feel like there are more than I would like already though. Maybe I can make a variadic macro or something.